src/common/cockpitjsonprint.c

```diff
@@ -28,6 +28,11 @@
 #include <string.h>
 #include <sys/mman.h>
 
+/* no-op fallback for old kernels */
+#ifndef MFD_NOEXEC_SEAL
+#define MFD_NOEXEC_SEAL 0
```

Comment:
I don't like this — it's not at all clear from the point of use (below) that when we write `MFD_NOEXEC_SEAL` that we'll be getting something other than that.  This sort of thing is only OK when you're defining the value that it would have had if the header were present.

Reply:
That's what the "no-op" is about. I know the value on x86_64 is `0x0008U`, but I don't dare to assume that it's the same on every architecture. Can we rely on this?

Reply:
For "new stuff" like this I think we can, but fair enough if you don't want to do that.  I'm fine with a different approach, but setting this thing is 0 is still a no-go for me, sorry.